server/filters/recipientTableRowsFilter.ts

```diff
@@ -14,7 +14,7 @@ export default function recipientTableRowsFilter(recipients: Array<Recipient>):
         text: recipient.prisonNumber ? recipient.prisonNumber : moment(recipient.prisonerDob).format('DD-MM-YYYY'), // Column 2 - Prison number or DOB
       },
       { text: recipient.prisonAddress.premise }, // Column 3 - Prison name
-      { html: '<a href="" class="govuk-link">Edit details</a>' }, // Column 4 - Edit recipient details link - TODO SLM-63
+      { html: `<a href="/barcode/edit-contact/${recipient.contactId}" class="govuk-link">Edit details</a>` }, // Column 4 - Edit recipient details link
       { html: `<a href="/barcode/review-recipients/remove/${idx}" class="govuk-link">Remove</a>` } // Column 5 - Remove recipient link
```

Comment:
Not one for now, but perhaps we should change the 'Remove' anchor and its corresponding controller method to take the contact ID rather than the index in the array. Removing by contactID feels much safer than removing by index from an array (even though its only removing from an in-memory array structure rather than making a rest call like Edit is) 
Just a thought

Reply:
Ah yes - sounds good